entc/gen/template/base.tmpl

```diff
@@ -71,6 +71,31 @@ func As(fn AggregateFunc, end string) AggregateFunc {
 	}
 {{ end }}
 
+// ValidationError returns when validating a field fails.
+type ValidationError struct {
+    Field string
```

Comment:
Do we need this? I'm fine with leaving this type just with `err`, since a `ValidationError` is returned for both edge and field validators.

Reply:
The `Unwrap` would allow to get the actual error message (coming from the underlying validate function), with a separate `Field`, it is now also possible to say exactly which field this error is for.

It is possible to get this out with a regexp (as the error message contains this), but seeing that the underlying error is accessible without regexp parsing (using Unwrap), it'd be great to be able to get the field out too without having to do that.

Reply:
I understand the use-case, but have problem with using edges as fields here.

What do you think about:

```go
type ValidationError struct {
    Field string // Field name; may be empty for edge errors.
    Edge string  // Edge name; may be empty for field errors.
    err error
}
```
Or:
```go
type ValidationError struct {
    Name string // Field or edge name.
    err error
}
```
WDYT? I prefer the second option.

Reply:
👍  on the second one. I'll change!

Reply:
Updated and re-ran generation.